examples/06-verif-manual/vm-006-pinched_cylinder.py

```diff
@@ -0,0 +1,520 @@
+   No. 3, 1973.
+
+Analysis Type(s):
+ - Static Analysis ``ANTYPE=0``
+
+Element Type(s):
+ - 4-Node Finite Strain Shell Elements (SHELL181)
+ - 8-Node Finite Strain Shell Elements (SHELL281)
+
+.. image:: ../../_static/vm6_setup.png
+   :width: 400
+   :alt: VM6 Pinched Cylinder Problem Sketch
+
+Material Properties
+ - :math:`E = 10.5 \cdot 10^6 psi`
+ :math:`Nu = 0.3125`
+
+Geometric Properties:
+ - :math:`l = 10.35  in`
+ - :math:`r = 4.953  in`
+ - :math:`t = 0.094  in`
+
+Loading:
+ - :math:`F = 100  lb`
+
+Analysis Assumptions and Modeling Notes:
+ - A one-eighth symmetry model is used. One-fourth of the load is applied
+   due to symmetry.
+
+"""
+# sphinx_gallery_thumbnail_path = '_static/vm6_setup.png'
+
+###############################################################################
+# Start MAPDL
+# ~~~~~~~~~~~
+
+from ansys.mapdl.core import launch_mapdl
+
+# Start mapdl and clear it.
+mapdl = launch_mapdl()
+mapdl.clear()
+
+# Enter verification example mode and the pre-processing routine.
+mapdl.verify()
+mapdl.prep7()
+
+
+###############################################################################
+# Define Element Type
+# ~~~~~~~~~~~~~~~~~~~
+# Set up the element type (a shell-type).
+
+# Type of analysis: Static.
+mapdl.antype("STATIC")
+
+# Define element number.
+elem_num = 1
+
+# Element type: SHELL181.
+mapdl.et(elem_num, "SHELL181")
+
+# Special Features are defined by keyoptions of shell element:
+
+# KEYOPT(3)
+# Integration option:
+# Full integration with incompatible modes.
+mapdl.keyopt(elem_num, 3, 2)  # Cubic shape function
+
+
+###############################################################################
+# Define Material
+# ~~~~~~~~~~~~~~~
+# Set up the material.
+
+# Define material number.
+mat_num = 1
+
+# Define material properties.
+mapdl.mp("EX", mat_num, 10.5E6)
+mapdl.mp("NUXY", mat_num, 0.3125)
+print(mapdl.mplist())
+
+
+###############################################################################
+# Define Section
+# ~~~~~~~~~~~~~~
+# Set up the cross-section properties for a shell element.
+
+# Parametrization of the shell thickness.
+t = 0.094
+
+# Define cross-section number.
+sec_num = 1
+
+# Define shell cross-section.
+mapdl.sectype(secid=sec_num, type_="SHELL", name="shell181")
+mapdl.secdata(t, mat_num, 0, 5)
+
+
+###############################################################################
+# Define Geometry
+# ~~~~~~~~~~~~~~~
+# Set up the keypoints and create the area through the keypoints.
+
+# Define global cylindrical coordinate system.
+mapdl.csys(1)
+
+# Define keypoints by coordinates.
+mapdl.k(1, 4.953)
+mapdl.k(2, 4.953, "", 5.175)
+
+# Generates additional keypoints from a pattern of keypoints.
+mapdl.kgen(2, 1, 2, 1, "", 90)
+
+# Create an area through keypoints.
+mapdl.a(1, 2, 4, 3)
+
+# Define the number of the keypoint where F is applied using inline function.
+q = mapdl.queries
+top_keypoint = q.kp(4.953, 90, 0)
```

Comment:
Best to simply access the attribute directly rather than creating a variable for it unless it's used more than once.
```suggestion
top_keypoint = mapdl.queries.kp(4.953, 90, 0)
```